node/libs/crypto/src/bn254/testonly.rs

```diff
@@ -1,33 +1,46 @@
 //! Random key generation, intended for use in testing
 
 use super::{AggregateSignature, PublicKey, SecretKey, Signature};
-use rand::{distributions::Standard, prelude::Distribution, Rng};
+use pairing::bn256::{Fr, G1, G2};
+use rand::{distributions::Standard, prelude::Distribution, Rng, RngCore};
+use rand04::Rand;
+
+struct RngWrapper<R>(R);
+
+impl<R: RngCore> rand04::Rng for RngWrapper<R> {
+    fn next_u32(&mut self) -> u32 {
```

Comment:
for efficiency implement next_u64 as well

Reply:
Fixed.